Plugin/CppApi/include/ToolResourceProvider.h

```diff
@@ -74,14 +75,20 @@ public slots:
   void onMousePressedAndHeld(QMouseEvent& mouseEvent);
   void onMouseDoubleClicked(QMouseEvent& mouseEvent);
   void onIdentifyGraphicsOverlayCompleted(QUuid id, IdentifyGraphicsOverlayResult* identifyResult);
+  void onIdentifyGraphicsOverlaysCompleted(QUuid id, QList<IdentifyGraphicsOverlayResult*> identifyResults);
+  void onIdentifyLayerCompleted(QUuid taskId, IdentifyLayerResult* identifyResult);
```

Comment:
should we keep the parameter consistently `id` throughout?

Reply:
i think taskId is more descriptive